application/models/Question.php

```diff
@@ -171,13 +171,10 @@ public function rules()
             /* ExpressionManager reserved word (partial) */
             $aRules[] = array('title', 'in', 'not' => true,
                 'range' => array(
-                    'LANG',
-                    'SID',
-                    'QID',
-                    'GID',
-                    'SAVEDID',
-                    'SGQ',
-                    'TOKEN',
+                    'LANG','SID', // Global var
+                    'SAVEDID','TOKEN', // current survey related var
+                    'QID','GID','SGQ', // current question related var
+                    'self','that','this', // EM reserved variables
```

Comment:
Does that mean that "self" as an answer code will not be accepted anymore?

Reply:
As a question title , not answer or Subquestion title

Reply:
Ahh, that makes sense. Being able to use it as answer text would be very helpful so if possible let's keep that.

Reply:
Yes, no issue in answer text : EM have issue with it currently, but it's an issue in EM.

But clearly : we must not allow it in question title/code